lib/extendOrdWithCustom.js

```diff
@@ -1,5 +1,5 @@
-const path = require("path");
-const cds = require("@sap/cds");
+const { root, log } = require("@sap/cds");
+const { join } = require("path");
```

Comment:
From my side, I would prefer `path.join`. When I read `join` independently, I need more context what is about, could be string.join, list.join and so on. `path.join` can provide the context, what do you think?